modules/restful_example/plugins/restful/taxonomy_term/tags/1.0/RestfulEntityTaxonomyTermTags.class.php

```diff
@@ -6,16 +6,4 @@
  */
 
 class RestfulEntityTaxonomyTermTags extends \RestfulEntityBaseTaxonomyTerm {
-
-  /**
-   * Overrides \RestfulEntityBaseTaxonomyTerm::checkEntityAccess().
-   *
-   * Allow access to create "Tags" resource for privileged users, as
-   * we can't use entity_access() since entity_metadata_taxonomy_access()
-   * denies it for a non-admin user.
-   */
-  protected function checkEntityAccess($op, $entity_type, $entity) {
-    $account = $this->getAccount();
-    return user_access('create article content', $account);
```

Comment:
I don't think we can remove this  - If I'm not mistaken it's used in one of RESTful's examples.

It's also probably needed to show people how the create for tags can be controlled (I don't have a better solution).
So it should become:

```php
protected function checkEntityAccess($op, $entity_type, $entity) {
  $account = $this->getAccount();
  return parent:: checkEntityAccess(..) && user_access('create article content', $account);
}
```